frontend/src/components/transactions/ReceiptRow.tsx

```diff
@@ -48,45 +66,229 @@ class ReceiptRow extends React.Component<Props> {
       );
     }
 
+    let gasBurnedByReceipt = new BN(0);
+    let tokensBurnedByReceipt = new BN(0);
+    let gasBurnedByConvertation = new BN(0);
+    let tokensBurnedByConvertation = new BN(0);
+    if (
+      receiptOutcomesById[receiptHash] &&
+      receiptOutcomesById[receiptHash].outcome
+    ) {
+      gasBurnedByReceipt = new BN(
+        receiptOutcomesById[receiptHash].outcome.gas_burnt
+      );
+      tokensBurnedByReceipt = new BN(
+        receiptOutcomesById[receiptHash].outcome.tokens_burnt
+      );
+    }
+    if (convertedReceipt) {
+      gasBurnedByConvertation = transaction.transactionOutcome
+        ? new BN(transaction.transactionOutcome.outcome.gas_burnt)
+        : new BN(0);
+      tokensBurnedByConvertation = transaction.transactionOutcome
+        ? new BN(transaction.transactionOutcome.outcome.tokens_burnt)
+        : new BN(0);
+    }
+
     return (
-      <Row noGutters className="receipt-row mx-0">
-        <Col className="receipt-row-details">
-          <Row noGutters>
-            <Col md="8" xs="7">
+      <Row
+        noGutters
+        className={
+          !convertedReceipt ? "receipt-row pl-4 mx-0" : "receipt-converted-row"
+        }
+        key={receiptOutcomesById[receiptHash].id}
+      >
+        <Col>
+          {convertedReceipt && (
+            <>
               <Row noGutters>
-                <Col className="receipt-row-title">{statusInfo}</Col>
+                <Col className="receipt-row-title receipt-hash-title">
+                  <b>Convert Transaction To Receipt</b>
+                </Col>
               </Row>
-              <Row noGutters>
-                <Col className="receipt-row-text">
-                  {receipt.outcome.logs.length === 0 ? (
-                    "No logs"
+
+              <Row noGutters className="receipt-row mx-0 pl-4">
+                <Col className="receipt-row-title">Gas Burned:</Col>
+                <Col className="receipt-row-receipt-hash">
+                  {gasBurnedByConvertation ? (
+                    <Gas gas={gasBurnedByConvertation} />
                   ) : (
-                    <pre>{receipt.outcome.logs.join("\n")}</pre>
+                    "..."
                   )}
                 </Col>
               </Row>
-            </Col>
-            <Col md="4" xs="5" className="ml-auto text-right">
-              <Row>
+
+              <Row noGutters className="receipt-row mx-0 pl-4">
+                <Col className="receipt-row-title">Tokens Burned:</Col>
                 <Col className="receipt-row-receipt-hash">
-                  {`${receipt.id.substr(0, 7)}...`}
+                  {tokensBurnedByConvertation ? (
+                    <Balance amount={tokensBurnedByConvertation.toString()} />
+                  ) : (
+                    "..."
+                  )}
                 </Col>
               </Row>
+            </>
+          )}
```

Comment:
This should be moved to a separate component (`TransactionOutcome`) and just used directly from `frontend/src/pages/transactions/[hash].jsx`, so there is no need to have `convertedReceipt` in this component.